src/dlgProfilePreferences.cpp

```diff
@@ -189,6 +189,8 @@ dlgProfilePreferences::dlgProfilePreferences(QWidget* pF, Host* pHost)
                                              "that symbol are listed, which may help to identify any unexpected or odd cases.<p>"));
     fontComboBox_mapSymbols->setToolTip(tr("<p>Select the only or the primary font used (depending on <i>Only use symbols "
                                            "(glyphs) from chosen font</i> setting) to produce the 2D mapper room symbols.</p>"));
+    fontComboBox_mapNames->setToolTip(tr("<p>Select the font used to show the 2D mapper room names.</p>"));
+    spinBox_mapNamesSizeAdj->setToolTip(tr("<p>Adjust the font size of the room names.</p>"));
```

Comment:
I thought the font size was being adjusted when it was drawn beneath the rooms - so why must the size be set here? :confused: